SPECS-EXTENDED/opencsd/opencsd.spec

```
@@ -0,0 +1,106 @@
+Summary:        An open source CoreSight(tm) Trace Decode library
+Name:           opencsd
+Version:        1.4.0
+Release:        0%{?dist}
```

We might want to bump the release :
```suggestion
Release:        1%{?dist}
```